src/PantherTestCaseTrait.php

```diff
@@ -239,4 +243,9 @@ private static function getWebServerDir(array $options): string
 
         return $_SERVER['PANTHER_WEB_SERVER_DIR'];
     }
+
+    private static function isGetClientStaticMethodAvailable(): bool
```

Comment:
This change doesn't look necessary. Can you please keep the diff as small as possible to help the review?

Reply:
I can revert this, but there will still be associated changes, as I replaced `\is_callable()` with `method_exists()` in order to resolve 2 phpstan errors that were/are present on `symfony/panther:main`

But if you want me to remove the function and deal instead with the two line changes instead, or accept the phpstan errors in which case I can fully refert to `\is_callable()`.  Let me know @dunglas

Reply:
I've removed this new method, but the remaining `\is_callable()` to `method_exists()` change remains to solve 2 phpstan errors.

@dunglas please resolve if sufficient